doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,292 @@ GAF has been implemented in Zephyr with the following structure.
+* C.3 Mandatory if the Hearing Aid Present Control Point characteristic is supported
+* C.4 Mandatory if the Hearing Aid Present Control Point characteristic and EATT are both supported
+* C.5 Mandatory if it supports characteristic values larger than the ATT_MTU
+
+Profile Dependencies
+====================
+
+Besides depending on GATT several of the LE Audio profiles depend on other profiles and services, as outlined in the following tables.
+
+.. table:: BAP dependencies
+   :widths: auto
+
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   |                    | Unicast Server | Unicast Client | Broadcast Source | Broadcast Sink | Scan Delegator | Broadcast Assistant |
+   +====================+================+================+==================+================+================+=====================+
+   | BAP Scan Delegator |                |                |                  | M              |                |                     |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   | ASCS Client        |                | M              |                  |                |                |                     |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   | ASCS Server        | M              |                |                  |                |                |                     |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   | PACS Client        |                | M              |                  |                |                | O                   |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   | PACS Server        | M              |                |                  | M              |                |                     |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   | BASS Client        |                |                |                  |                |                | M                   |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+   | BASS Server        |                |                |                  |                | M              |                     |
+   +--------------------+----------------+----------------+------------------+----------------+----------------+---------------------+
+
+Notes:
+* As the table shows, the Broadcast Source role has no dependencies on other LE Audio profiles or services
+
+.. table:: CAP dependencies
+   :widths: auto
```

Comment:
I think the order can be improved. The LE Audio stack is a layered architecture, and the tables should follow the layers. 

So either put HAP/TMAP/GMAP/etc. at the top followed by CAP followed by the lower layer profiles, or vice versa. Right now it seems mixed